app/assets/stylesheets/partner.scss

```diff
@@ -1,3 +1,8 @@
 .map-marker {
 	height: 300px !important;
 }
+
+.breadcrumb .size-patch {
```

Comment:
should probably be located with the rest of the breadcrumb styles, I thought I'd fixed the uneven buttons sorry if not!

https://github.com/geeksforsocialchange/PlaceCal/blob/main/app/components/breadcrumb/breadcrumb.sass

Reply:
That's a good idea but this is not a global fix for the entire site and is defined in a class that is only used in this one form.

Unless this filter pattern is going to be used site-wide?

Reply:
ok, you've convinced me, I think mark will redesign this at some point so we should just remember to remove it then.